templates/taxi/car_list.html

```diff
@@ -7,11 +8,17 @@ <h1>
       Create
     </a>
   </h1>
+
+  <form action="" method="get" class="form-inline mb-3">
+    {{ search_form|crispy }}
+    <input class="btn btn-outline-secondary" type="submit" value="🔎">
+  </form>
+
```

Comment:
What about separating this form into some separate file and just include search where it's needed instead of repeating the code?